### ISSUE TITLE
Adding options

### ISSUE THREAD

**Issue by zweahtet** [hckr/fb-post-screenshot#35]
I like this project. Recently, I was looking for an extension that can save the contents and pictures as a pdf file or an img from a facebook post. I came across your GitHub project. Just my thought. I was thinking whether it is possible to save just the contents and pictures as a pdf file or an img. Is there any way to opt out saving the comments? Thanks!

**Comment by hckr** [comment]
Yes, there is such an option (to include just the post). Please see the add-on options.

**Comment by zweahtet** [comment]
> Yes, there is such an option (to include just the post). Please see the add-on options.

oh. I found it. Thanks!
